packages/ai-jsx/src/core/completion.tsx

```diff
@@ -29,7 +29,7 @@ function AutomaticCompletionModel({ children, ...props }: ModelPropsWithChildren
   }
 
   throw new Error(
-    'No completion model was specified. Specify a CompletionProvider or set the OPENAI_API_KEY environment variable.'
+    'No completion model was specified. Set the OPENAI_API_KEY environment variable to use OpenAI or use an explicit CompletionProvider.'
   );
```

Comment:
It doesn't have to be in this PR, but eventually I'd like all our errors to have an `AI.JSX(1234)` error code, for easy googling / referencing from docs / etc.